src/Inkluzitron/Services/UserBdsmTraitsService.cs

```diff
@@ -143,6 +143,12 @@ public async Task<BdsmTraitOperationCheck> CheckDomSubOperationAsync(IUser user,
                 return check;
             }
 
+            if (userDb.Points < 0)
```

Comment:
Tohle nebude fungovat po mergi #101, takže bych tohle margnul až po té změně bodů.

Reply:
A nebo teda prvně mergneme tohle a já to fixnu u sebe

Reply:
Tez si myslim 😆